packages/medusa/src/utils/clean-response-data.ts

```diff
@@ -1,21 +1,53 @@
 import { pick } from "lodash"
 
+// TODO: once the legacy totals decoration will be removed.
+// We will be able to only compute the totals if one of the total fields is present
+// and therefore avoid totals computation if the user don't want them to appear in the response
+// and therefore the below const will be removed
+const EXCLUDED_FIELDS = [
```

Comment:
question: is this the right place for this? This should be coming from the place where the route lives.

Reply:
It is applicable for everything and is meant to be temporary